### ISSUE TITLE
Over-indented code generated in ast header file

### ISSUE THREAD

**Issue by iomaganaris** [BlueBrain/nmodl#327]
I noticed that some code blocks inside the generated `ast.hpp` file get over-indented. For example:
```
/**
 * \brief Return name of of the node
 *
 * Some ast nodes have a member marked designated as node name. For example,
 * in case of this ast::Name has name designated as a
 * node name.
 *
 * @return name of the node as std::string
 *
 * \sa Ast::get_node_type_name
 */
virtual std::string get_node_name() override {
    return name->get_node_name();
}
                   /**
                    * \brief Getter for member variable \ref ThreadsafeVar.name
                    */
                   std::shared_ptr<Name> get_name(){
                       return name;
                   }
        /// \}
        /// \name Setters
        /// \{
        /**
         * \brief Set token for the current ast node
         */
        void set_token(ModToken& tok) { token = std::make_shared<ModToken>(tok); }
                       /**
                        * \brief Setter for member variable \ref ThreadsafeVar.name (rvalue reference)
                        */
                       void set_name(std::shared_ptr<Name>&& name) {
                           this->name = name;
                       }
                       /**
                        * \brief Setter for member variable \ref ThreadsafeVar.name
                        */
                       void set_name(const std::shared_ptr<Name>& name) {
                           this->name = name;
                       }
        /// \}
```
Most if not all of those wrongly indented blocks are `getters` and `setters` as far as I've seen

**Comment by cattabiani** [comment]
may you check that this was not automatically solved by #312 ?

**Comment by iomaganaris** [comment]
It's true. This issue is fixed right now, so we may close the ticket
